### PR TITLE
Add (Simplified) Chinese Localization

### PR DESCRIPTION
**Pull request by diaowinner**
Add Simplified Chinese localization of PinkSea.

### CODE REVIEW

**Comment by diaowinner** [comment]
One problem is that I used to use the "zh-CN" tag instead of "zh" (there's Traditional Chinese to support, too!), but I don't know how to do.

File: PinkSea.Frontend/src/intl/i18n.ts [hunk 1]
```diff
@@ -8,46 +8,51 @@ import It from '@/intl/translations/it'
...
   },
   bg: {
-    name: "български",
-    translation: Bg
-  }
-};
+    name: 'български',
+    translation: Bg,
+  },
+  zh: {
```
Comment (purifetchi) [inline PinkSea.Frontend/src/intl/i18n.ts:54]: you should be able to do `"zh-CN"` in here to specify that this is simplified chinese :)

**Comment by purifetchi** [comment]
this is wonderful work, thank you!!

i've only given some ideas on how to better mark that this is simplified chinese, to then support traditional if anyone ever offers to do it! ^^

File: PinkSea.Frontend/src/intl/i18n.ts (hunk 1 above)
Comment (diaowinner) [reply PinkSea.Frontend/src/intl/i18n.ts:54]: Yes, that's what I want to.  
But when I renamed, the error occurs saying that you shouldn't use hyphen as tag name, and I'm lazy, so I changed it to `zh`.

I can hardly use the computer. If there's any spare time I'll give it another shot.

Comment (purifetchi) [reply PinkSea.Frontend/src/intl/i18n.ts:54]: if you want, i can merge your changes as-is and give it a shot myself to rename it to "zh-CN" :)

Comment (diaowinner) [reply PinkSea.Frontend/src/intl/i18n.ts:54]: Sure. If there's any bug please fix it.

**Comment by purifetchi** [comment]
thank you so much once again!!!

**Pull request merged**